pkgs/development/tools/continuous-integration/buildbot/plugins.nix

```diff
@@ -29,6 +29,42 @@
     };
   };
 
+  www-react = buildPythonPackage rec {
+    pname = "buildbot-www-react";
+    inherit (buildbot-pkg) version;
+    format = "wheel";
+
+    # fetchpypy returns a 404 for the wheel?
+    # normal source release doesn't have any assets
+    src = fetchurl {
+      url = "https://github.com/buildbot/buildbot/releases/download/v${version}/buildbot_www_react-${version}-py3-none-any.whl";
+      hash = "sha256-5+uPGAGJcfwev8bh3O1NSmZFcokr7pzBiAyQg8c8TLA=";
+    };
+
+    # Remove unneccessary circular dependency on buildbot
+    postPatch = ''
+      pushd dist
+      unzip buildbot_www_react-${version}-py3-none-any.whl
+      sed -i "s/Requires-Dist: buildbot//" buildbot_www_react-${version}.dist-info/METADATA
+      chmod -R u+w buildbot_www_react-${version}-py3-none-any.whl
+      zip -r buildbot_www_react-${version}-py3-none-any.whl buildbot_www_react-${version}.dist-info
+      popd
+    '';
+
+    buildInputs = [ buildbot-pkg ];
+    nativeBuildInputs = [ unzip zip ];
+
+    # No tests
+    doCheck = false;
+
+    meta = with lib; {
+      homepage = "https://buildbot.net/";
+      description = "Buildbot UI (React)";
+      maintainers = with maintainers; [ ryansydnor lopsided98 ];
```

Comment:
Did @ryansydnor @lopsided98 agree to maintain this or did you forget to remove them?

Reply:
lopsided98 wanted to add it in an older PR but it still had some bug their. All these plugins are maintained by buildbot, so it makes sense to have the maintainer of buildbot also maintaining these one.s

Reply:
Added myself instead.